modules/virt-NUMA-live-migration.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * virt/managing_vms/advanced_vm_management/virt-NUMA-topology.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-NUMA-live-migration_{context}"]
+= Live migration outcomes using vNUMA
+
+[role="_abstract"]
+Migration outcomes for VMs are dependent on the configured Topology Manager policies.
+These policies determine how CPU and memory resources are allocated with respect to the physical NUMA nodes of the host.
+There are four available policies: `None`, `single-numa-node`, `best-effort`, and `restricted`.
+
+The following table outlines which policies are supported for different VM configurations, and their effect on live migration.
+
+* A small VM is defined as a VM with less total cores than half of cores in NUMA node.
+* A large VM is defined as a VM with more total cores than half of cores in NUMA node.
+* An extra large VM is defined as a VM with more cores than 1 NUMA node.
+
+[cols="3",options="header"]
+|===
+|VM configuration
+|Topology Manager policy
+|Tested support status
+
+|Small
+|None
+|Live migration works, as expected.
+
+|Small
+|single-numa-node
+|Live migration works, as expected.
+
+|Small
+|best-effort
+|Live migration works, as expected.
+
+|Small
+|restricted
+|Live migration works, as expected.
+
+|Large
+|None
+|Live migration does not work. This is a known issue. The process ends with an incorrect memnode allocation error, and libvirt rejects the XML manifest generated by KubeVirt. See release notes for additional information.
```

Comment:
@abrennan89 This can happen with any VM irrespective of size. This is basically capacity issues on available worker nodes as they lack capacity to allocate enough memory due to mode restrictions. I suggest adding this comment as side note and not limiting it based on large or small VM 
cc @ksimon1